Framework/DataHandling/src/LoadEventAsWorkspace2D.cpp

```diff
@@ -90,6 +90,9 @@ void LoadEventAsWorkspace2D::init() {
                   "The name of the units to convert to (must be one of those registered in the Unit Factory)");
   declareProperty(std::make_unique<WorkspaceProperty<Workspace2D>>("OutputWorkspace", "", Direction::Output),
                   "An output workspace.");
+  declareProperty("LoadNexusInstrumentXML", true,
+                  "If true, attempt to load the instrument from the NeXus file. "
+                  "If false, only attempt to load the local copy of the instrument XML.");
```

Comment:
I'd rather say:
 "If true, load the instrument definition file (IDF) from the input NeXus file. If false, Mantid will load the most appropriate  IDF from the instrument repository"

Reply:
Done